tests/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -367,25 +367,9 @@ class PublicSuite extends TreeSuiteBase {
     // n/a
   }
 
-  test("scala.meta.tokenizers.TokenizeException.toString") {
-    interceptMessage[TokenizeException](
-      """|<input>:1: error: unclosed string literal
-         |"c
-         |^""".stripMargin.lf2nl
```

Comment:
Should we instead make sure that invalid tokens are produced? Either that or remove the tests.

Reply:
we can't remove these tests, without removing the entire PublicSuite. why it exists is a bit unclear for me, it is not to test functionality but to ensure that public methods are somehow mentioned here explicitly...

the actual functionality should be well tested in TokenizerSuite and similar ones. i remove the body of these tests because it no longer worked as before.